src/MigrationsGeneratorServiceProvider.php

```diff
@@ -80,6 +80,9 @@ function ($app) {
     public function boot()
     {
         if ($this->app->runningInConsole()) {
+            $this->publishes([
```

Comment:
Please remove `publishes`. Will implement it next time when we have better way to `setStubPath`.